packages/wonder-blocks-link/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "wonder-blocks-button",
```

Comment:
I saw the error you were getting - I wonder if you need to update this name? (I see that the dependencies are correct below, so I'm not sure what else it could be.